packages/astro/src/core/dev/dev.ts

```diff
@@ -36,9 +35,6 @@ export default async function dev(
 		inlineConfig,
 		logging: options.logging,
 		fs,
-		handleConfigError: options.handleConfigError,
-		// eslint-disable-next-line no-console
-		beforeRestart: () => console.clear(),
```

Comment:
The `beforeRestart` was only created because on `console.clear()`, as we don't want to call `console.clear` in tests (test results gets removed).

Personally I find clearing the console moves and cuts out the terminal a lot that it's better to not do it instead, so I removed it.

Reply:
Totally agree, especially if there are scripts/users that rely on stdout/stderr